src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.label {
+  display: block;
+  position: relative;
+}
+
+.labelBig::before {
+  content: "";
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-image: url(images/Search.svg);
+}
+
+.labelSmall::before {
+  content: "";
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-image: url(images/Search.svg);
+  background-size: cover;
+}
+
+.search {
+  box-sizing: border-box;
+  width: 100%;
+  display: block;
+  font-weight: 400;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: all 0.5s ease;
+}
+
+.search::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  transition: all 0.5s ease;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+  font-weight: bold;
+  outline: none;
+}
+
+.bigSearch {
+  height: 70px;
+  padding: 0 17px 0 62px;
```

Comment:
I guess if you used verical paddings you could avoid hardcoding the heigth

Reply:
> I guess if you used verical paddings you could avoid hardcoding the heigth

I made changes but now I fail github checks. Although npm test works fine on my computer, github tells me that all checks have failed. Can You explain me why?